gptme/config.py

```diff
@@ -242,11 +242,12 @@ class ChatConfig:
     tool_format: "ToolFormat | None" = None
     stream: bool = True
     interactive: bool = True
+    workspace: Path = field(
+        default_factory=Path.cwd
+    )  # TODO: Is default value cwd ok for server?
```

Comment:
`workspace` should be retrieved from the `$logdir/workspace` symlink/folder. Right now we store it both in the config and as the logdir folder/symlink, and they will get out of sync.